style.css

```diff
@@ -317,12 +345,13 @@ body {
   line-height: 16px;
   color: #ffffff;
   position: absolute;
-  top: 140px;
+  top: calc(100% - 20px);
```

Comment:
We use a calc now because screen share tiles are of a different size than camera tiles, so the height is no longer consistent.